packages/product-react/src/components/Radio/Radio.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import clsx from 'clsx';
+
+export interface IRadio extends React.HTMLProps<HTMLInputElement> {
+  className?: string;
+  id: string;
+  label?: string;
+  name?: string;
+  pill?: boolean;
+}
+
+export const Radio: React.FC<IRadio> = ({
+  className,
+  id,
+  name,
+  label,
+  pill,
+  ...rest
+}) => {
+  const wrapperClass = clsx(
+    {
+      'ray-radio': !pill,
+      'ray-radio-pill': pill
+    },
+    className
+  );
+
+  return (
+    <div className={wrapperClass}>
+      <input
+        className={pill ? 'ray-radio-pill__input' : 'ray-radio__input'}
```

Comment:
should this class be `ray-radio__pill-input`?

Reply:
nope ,https://ray.wework.com/components/radio/